actors/builtin/miner/policy.go

```diff
@@ -83,7 +83,7 @@ var MaxSealDuration = map[abi.RegisteredSealProof]abi.ChainEpoch{
 
 // Number of epochs between publishing the precommit and when the challenge for interactive PoRep is drawn
 // used to ensure it is not predictable by miner.
-var PreCommitChallengeDelay = abi.ChainEpoch(150)
+var PreCommitChallengeDelay = abi.ChainEpoch(125) // 1 hour
```

Comment:
This used to be 62.5 minutes.

Reply:
This must be 150 - it has nothing to do with time, it has to do with number of blocks.